### ISSUE TITLE
Can't use mocha_test on non-root packages

### ISSUE THREAD

**Issue by driveraid** [pubref/rules_node#37]
Hi @pcj, I think I found an issue with mocha_test. It appears it doesn't work for packages deeper than the workspace root.

To reproduce, simply:
```
$ cd tests/mocha
$ mkdir W
$ mv BUILD test.js W
$ bazel test //...
```
It appears the mocha launcher doesn't take the leading path into account. Please see the attachment for a solution which I verified is working both for the base package and deeper ones.

[0001-Make-sure-mocha_test-can-be-used-on-other-packages-t.patch.txt](https://github.com/pubref/rules_node/files/1375349/0001-Make-sure-mocha_test-can-be-used-on-other-packages-t.patch.txt)

**Comment by cwoodcock** [comment]
Just ran into this using head @ f990afc.  Seems #41 accidentally added a regression.  Backing the truck up to dc9f8ba worked around it for me.  cc @pcj

**Comment by pcj** [comment]
Shucks, sorry about that.  #41 was a big change.  I'll take a look at this tonight.

**Comment by pcj** [comment]
Should be fixed by #42
